components/builder-admin-proxy/hooks/health_check

```diff
@@ -5,7 +5,8 @@ set -x
 # default return code is 0
 rc=0
 
-hab pkg exec core/curl curl --head --fail --max-time 1 http://localhost
+hab pkg exec core/curl curl --head --fail --max-time 1 \
```

Comment:
Yeah this was there before, but it's a super easy fix and definitely what we should be doing. I just made this change and pushed it up!